kubernetes/zenko/values.yaml

```diff
@@ -86,6 +86,16 @@ cloudserver:
       # secret:
 
 backbeat:
+  api:
+    autoscaling:
+      enabled: true
+      config:
+        maxReplicas: *nodeCount
```

Comment:
I don't really get why this should be `nodeCount`. We use single-process NodeJS containers, right? So the number of processes you could in theory run is the number of CPUs in the cluster, which is likely (much) more than the number of nodes in the cluster.

I'm not sure what the  'correct' number here should be, or how to calculate it. However, `nodeCount` seems at least as arbitrary as 3, 5, 10 or 21...

Reply:
Since we are basing all of our default replicas in Zenko off `nodeCount` it isn't arbitrary at all to continue that nomenclature here. It's the safest default we can assume without bottlenecking at a single pod/process while at the same time allowing anyone to change the value based off their core count if they so desire.

Additionally, this is an internal-only facing service at the moment so all the load that it receives is from our other processes. This is how we know that typically 1 pod/process is enough, however under heavy CRR operations, for example, we also know that 1 pod/process can become exhausted therefore tripling the capacity (for a 3 node) can be a safe assumption.

Reply:
Again that's why I think the hpa is a better solution than having a static replica count here. Because if the cluster sees that a single pod is near the designated capacity, then it can add pods as necessary to fill the demand.